lib/provider/global_provider.dart

```diff
@@ -509,6 +510,38 @@ class GlobalProvider with ChangeNotifier {
     notifyListeners();
   }
 
+  setDisabledLanguages(Map<String, bool> value) {
+    _disabledLanguageMap = value;
+    notifyListeners();
+  }
+
+  Future<bool> isFilePresent(String filePath) async {
+    try {
+      await rootBundle.load(filePath);
+      return true;
+    } catch (e) {
+      return false;
+    }
+  }
+
+  setDisabledLanguage(String langCode) async {
+    String code = getTwoLanguageCode(langCode);
+    String filepath = "assets/l10n/app_$code.arb";
+    bool isPresent = await isFilePresent(filepath);
+    _disabledLanguageMap[langCode] = !isPresent;
+    notifyListeners();
+  }
+
+  getTwoLanguageCode(String code) {
```

Comment:
can we load this as a Map<> preferably from configuration and other languages as well, since this will mostlikely not change

Reply:
Added new constants file for adding locale of the languages.